### PR TITLE
Add NixOS install instructions

### PR DESCRIPTION
**Pull request by saviola777**
@Gurjaka and @Sigmanificient were nice enough to get qtile-bonsai added to nixpkgs, I'm proposing to add install instructions for NixOS to the `README.md` with this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,21 @@ your favorite AUR-helper. For example:
 yay -S qtile-bonsai
 ```
 
+#### NixOS
+
+It is available in [nixpgks](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=qtile-bonsai) as `qtile-bonsai`:
+
+```
```
Comment (Sigmanificient) [inline README.md:93]:
```suggestion
```nix
```

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

**Comment by aravinda0** [comment]
Oh, sweet!

Could you please also add this text to the `templates/README.template.md` file? 

That's the actual source of truth for the README, which is then generated with `python scripts/generate_readme.py`.
The docs and all are generated from docstrings in the code and put into the final readme. 

(In this case, you don't necessarily need to run the script and leaving the manual README edit is also okay, since the script will generate the same edits - as it's static text)

**Comment by saviola777** [comment]
Done!

**Comment by aravinda0** [comment]
Merged!

Thanks to everyone involved in handling the proper packaging into nixpkgs! Nix is still something I've only mildly dabbled in, so this would have taken me a lot longer to figure out.

**Pull request merged**